ui/page/governance/proposals_page.go

```diff
@@ -383,26 +385,29 @@ func (pg *ProposalsPage) layoutStartSyncSection(gtx C) D {
 }
 
 func (pg *ProposalsPage) listenForSyncNotifications() {
+	pg.ProposalNotificationListener = listeners.NewProposalNotificationListener()
```

Comment:
I think it's a good idea to abort if this listener isn't currently nil, to prevent duplicate listeners. Also, set the listener to nil after listening is done.